lib/Alien/Build/Interpolate.pm

```diff
@@ -167,10 +167,18 @@ sub execute_helper
 
 =head2 interpolate
 
+ my $string = $intr->interpolate($template, $build);
  my $string = $intr->interpolate($template);
 
-This takes a template and fills in the appropriate values of any helpers
-used in the template.
+This takes a template and fills in the appropriate values of any helpers used
+in the template.
+
+[version 2.58]
+
+If you pass in an L<Alien::Build> instance as the second argument, you can use
+properties as well as helpers in the template.  Example:
+
+ my $patch = $intr->template("%{.install.patch}/foo-%{.runtime.version}.patch");
```

Comment:
The example doesn't show using the second argument to `interpolate`, it shows use of a `template` method taking a single argument.

Reply:
Thanks for catching that.  Fixed in #341.